content/docs/index.md

```diff
@@ -19,7 +23,7 @@ improve this [documentation](https://github.com/iterative/dvc.org) 🙏.
 ## Main topics
 
 - Learn how to [install](/doc/install) and
-  [get started](/doc/tutorials/get-started) with DVC in the first sections.
+  [get started](/doc/tutorials/get-started) with DVC in the first few docs.
 - Explore the main [reasons](/doc/use-cases) to adopt DVC.
 - Study the detailed [inner-workings](/doc/user-guide) of DVC, as well as each
   one of its [commands](/doc/command-reference) and Python
```

Comment:
@shcheklein what did you mean by

> main topics - need a design with tiles here (can do HTML in Markdown for example)

Reply:
OK here the suggestion is to have a special UI design for example with boxes or tiles that link to each main topic.Probably responsive. Something more attractive but in effect like a table.

Reply:
Hmmm some basic HTML breaks the markdown parsing it seems, e.g.:

```markdown
## Main topics

<div>
  Learn how to [install](/doc/install) and
  [get started](/doc/tutorials/get-started) with DVC in the first few docs.
</div>
```

Looks like:

![image](https://user-images.githubusercontent.com/1477535/79535465-30e99500-8043-11ea-826a-e8c6dd08e708.png)

Reply:
I really have no idea how this is currently setup but maybe this has something to do? https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-transformer-remark#excerpts-for-html-embedded-in-markdown-files

Cc @pavelgrinchenko any hints would be appreciated 🙂

Reply:
OK I created some basic tiles with pure HTML/CSS @shcheklein just to give us an idea but this clearly needs I work 😆 

See https://dvc-landing-refactor-ge-etwyy1.herokuapp.com/doc/home

Reply:
I'm not sure I can help now. Problem need investigation....